jill/install.py

```diff
@@ -15,6 +15,32 @@
 import subprocess
 
 
+def _get_relative_bin_path():
+    system = current_system()
+    if system == "winnt":
+        return os.path.join("bin", "julia.exe")
+    elif system == "mac":
+        return os.path.join("Contents", "Resources", "julia", "bin", "julia")
+    elif system == "linux" or system == "freebsd":
+        return os.path.join("bin", "julia")
+    else:
+        raise ValueError(f"Unsupported system {system}")
+
+
+def get_installed_bin_paths():
+    """
+    Return a dict whose keys are Julia version strings and whose values are
+    absolute paths to the corresponding Julia exectuable. Only jill.py's
+    default directory for Julia installations is searched.
+    The version strings either equal to `latest` or have the form `major.minor`.
```

Comment:
If my memory serves well, there can be a third case `julia-dev` if you install the dev version Julia via `jill install 1.8.0+848d273`.